cluster/gce/trusty/master.yaml

```diff
@@ -216,7 +216,6 @@ script
 	export HOME="/root"
 	export KUBECTL_BIN="/usr/bin/kubectl"
 	export TOKEN_DIR="/etc/srv/kubernetes"
```

Comment:
I notice PR #23603 removed the usage of "TOKEN_DIR". You can remove this export too